initfiles/bash/etc/systemd/system/thorslave@instance.service.in

```diff
@@ -6,10 +6,12 @@ After=network.target remote-fs.target @dependencies@
 Type=simple
 User=@RUNTIME_USER@
 Group=@RUNTIME_GROUP@
-PermissionsStartOnly=true
+PermissionsStartOnly=false
```

Comment:
is this related?

Reply:
This one should be in here.  The one in another PR is regarding the thor@instance.service.in file I believe, but I'll double check.